packages/zosfiles/src/api/methods/create/Create.ts

```diff
@@ -134,6 +136,34 @@ export class Create {
         }
     }
 
+    public static async dataSetLike(session: AbstractSession, dataSetName: string, options: ICreateDataSetLikeOptions): Promise<IZosFilesResponse> {
+        const tempOptions = JSON.parse(JSON.stringify(options));
+
+        // Required
+        ImperativeExpect.toNotBeNullOrUndefined(dataSetName, ZosFilesMessages.missingDatasetName.message);
+
+        try {
+            const endpoint: string = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_DS_FILES + "/" + dataSetName;
+
+            const data = await ZosmfRestClient.postExpectString(session, endpoint, [], JSON.stringify(tempOptions));
+
+            let response = "";
+            if (!isNullOrUndefined(tempOptions.showAttributes)) {
+                delete tempOptions.showAttributes;
```

Comment:
Deleting `showAttributes` property from `tempOptions` object should be done sooner, before calling `ZosmfRestClient`.